.github/workflows/build_test_fmt.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       env:
         RUSTFLAGS: "-D warnings" # Trigger an error on warnings
       run: |
-        cargo clippy
+        cargo check
```

Comment:
This is good - it prevents us from merging if there are build warnings, but not clippy warnings

Reply:
Yes I agree